tests/testsuite.at

```diff
@@ -275,6 +275,8 @@ dnl Defines all possible Souffle flag configurations for testing.
 dnl NOTE: This is the default configuration that can be overridden
 dnl using SOUFFLE_CONFS environment variable.
 m4_define([DEFAULT_CONFS], [[], dnl interpreter
+ [-t0],                 dnl provenance
```

Comment:
Did you mean to add these back in? I think the provenance category of tests is sufficient.